### DISCUSSION TITLE
Override Theme of Activity Pop Up

### DISCUSSION THREAD

**Discussion by makattar** [elsa-workflows/elsa-core#1263] [Q&A]
Hi ,
Can I able to override theme of Elsa activity Component ?
Like i want to use material design there.How can i able to do that?

**Comment by sfmskywalker** [comment]
You might be able to do it by targeting the HTML elements and CSS classes it uses. If you run into an HTML element that you can't easily target e.g. because it doesn't have a semantic class name, then feel free to submit a PR that includes this.
